app/shared/src/commonMain/kotlin/ui/subject/episode/EpisodeVideo.kt

```diff
@@ -204,7 +207,14 @@ internal fun EpisodeVideoImpl(
                 )
             }
         },
-        rhsBar = {
+        rhsBarTop = {
+            if (expanded && Platform.currentPlatform.isDesktop()) {
```

Comment:
```suggestion
            if (expanded && currentPlatform.isDesktop()) {
```
目前在逐步淘汰 `Platform.currentPlatform`